### PR TITLE
Fixing dead link and adding more description about the build

### PR DESCRIPTION
**Pull request by mbruzek**
Replacing dead link and adding more information on the build process of the layers.

### CODE REVIEW

**Comment by deanmaniatis** [comment]
That looks great, thanks for the the improvement.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,36 @@
...
+layer you just created.
+
+```
+charm build
+```
+
+Now you should be able to deploy the assembled charm.
+
+```
+juju deploy local:<series>/<charm-name>
```
Comment (chuckbutler) [inline README.md]: local: charms are going away. this needs to read:

juju deploy ./trusty/docker

File: README.md [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
+base layer for other Charms that use docker. Please refer to the
+[Charm Layers](https://jujucharms.com/docs/devel/developer-layers)
+documentation.
 
 ## Usage
 
-In a charm that wants to use docker, the integration can be as simple as placing
-the following in your charm's `compose.yaml`:
+In a layer that wants to use docker, the integration can be as simple as
+placing the following in the `compose.yaml` file:
```
Comment (chuckbutler) [inline README.md]: s/compose.yaml/layer.yaml

**Comment by chuckbutler** [comment]
@mbruzek couple of comments left inline

File: README.md (hunk 2 above)
Comment (mbruzek) [inline README.md]: Doh! I can't believe I missed this!

Comment (deanmaniatis) [inline README.md]: This is correct, compose.yaml will not work.

**Comment by chuckbutler** [comment]
+1 LGTM

**Pull request merged**